testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.min_max_length_03/min.max.length.jst

```diff
@@ -0,0 +1,11 @@
+JSIGHT 0.3
```

Comment:
Please, add tests, where `minLength` and `maxLength` are in the same set of rules.

Reply:
fixed https://github.com/jsightapi/jsight-api-core/pull/15/commits/c575a57a5f0981d31c707d6a9dd806659738c94f